src/main/java/org/broadinstitute/hellbender/utils/variant/writers/HomRefBlock.java

```diff
@@ -164,7 +165,7 @@ public int getMinDP() {
     /** Get the median DP observed within this band
      * If there are an even number of DPs recorded in this band the median is the mean of the two middle values */
     public int getMedianDP() {
-        return (int) Math.round(MathUtils.median(DPs));
+        return (int) Math.round(new Median().evaluate(DPs.stream().mapToDouble(x->x).toArray()));
```

Comment:
done.  Previous implementation also used a stream, so I don't think an HC run is needed, but I'll be doing one anyway to test `MathUtils::applyToArray`.